src/config.js

```diff
@@ -17,6 +19,8 @@ export const plugins = [
 	Notes,
 	Admin,
 	Meta,
-	NetworkAdmin,
-	Fbw,  // fbw does not have menu item
+	// GroundRouting,
```

Comment:
We should not removed NetworkAdmin plugin neither add the comments related to GroundRouting back again